test/SILOptimizer/existential_specializer_indirect_class.sil

```diff
@@ -8,74 +8,68 @@ protocol ClassProtocol: AnyObject { func method() }
 
 class C: ClassProtocol { func method() {} }
 
-// CHECK-LABEL: sil shared @$s28test_indirect_class_protocolTf4e_n : $@convention(thin) <τ_0_0 where τ_0_0 : ClassProtocol> (@in τ_0_0) -> ()
-sil hidden @test_indirect_class_protocol : $@convention(thin) (@in ClassProtocol) -> () {
+// CHECK-LABEL: sil [signature_optimized_thunk] [always_inline] @test_indirect_class_protocol : $@convention(thin) (@in ClassProtocol) -> ()
```

Comment:
cc @jckarter looks like you wrote this test so, I should probably ping you before completely changing it. Does this update seem reasonable?

Reply:
I don't remember this test particularly, but it looks like the existential specializer is no longer applying to this function. Maybe that's expected because your changes are devirtualizing away uses of the existential to begin with? It'd be good to double check.

Reply:
Thanks for pointing that out. The specializations are still being generated but after they're inlined they are removed. I'm checking the wrong function here. I'll update the `RUN` command to exclude dead-function-elimination and then check the specialized function instead.